assets/ovirt.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: thin
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+provisioner: kubernetes.io/ovirt-csi
```

Comment:
`kubernetes.io` prefixes are reserved for Kubernetes and we expect that they're in-tree. You should use something else.

Reply:
Just to be sure, this is the name of the CSI driver too right? Provisioner requires that driver name should be same as provisioner name here.